recipe/meta.yaml

```
@@ -11,35 +11,37 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  skip: true # [py<38 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
   entry_points:
     - voila = voila.app:main
 
 requirements:
   host:
-    - python >=3.8
+    - python
     - pip
-    - nodejs >=18,<19
-    - hatch-jupyter-builder >=0.3.2
+    - hatch-jupyter-builder >=0.5.3
     - hatchling >=1.8.1
   run:
-    - python >=3.8
+    - python
     - nbconvert >=6.4.5,<8
     - jupyter_core >=4.11.0
     - jupyterlab_server >=2.3.0,<3
     - jupyter_client >=7.4.4,<=9
```

```suggestion
    - jupyter_client >=7.4.4,<9
```
Is v9 compatible? see https://github.com/voila-dashboards/voila/blob/6418e2521f6901fa896448502f9a36743b8747ce/pyproject.toml#L38

---

nope, thank you!